src/app/admin/store/reducers/admin-order.reducer.ts

```diff
@@ -0,0 +1,12 @@
+import { createReducer, on } from '@ngrx/store';
+
+import * as AdminPageOrderActions from '../actions/admin-order.actions';
+import { initialAdminOrdersState } from '../state/admin-orders.state';
+
+export const adminOrdersReducers = createReducer (
+  initialAdminOrdersState,
+  on(AdminPageOrderActions.GetOrdersSuccess,
```

Comment:
let's better use pattern from the docs (https://ngrx.io/guide/store/walkthrough)
should like this
```
export const booksReducer = createReducer(
  initialState,
  on(retrievedBookList, (state, { Book }) => [...Book])
);
```

will be much more readable

Reply:
fixed